src/components/Table.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+function Table({ list }) {
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>Currency</th>
+          <th>Value</th>
+          <th>Change</th>
+        </tr>
+      </thead>
+      <tbody>
+        {list.map(({ names, value, change }) => (
+          <tr key={names + value + change}>
+            <td key={names} className="table-element">
+              {names}
+            </td>
+            <td key={names + value} className="table-element">
+              {value}
+            </td>
+            <td key={names + change} className="table-element">
+              {change}
+            </td>
+          </tr>
+        ))}
+      </tbody>
+    </table>
+  );
+}
```

Comment:
This component is too smart, it should be dumb and agnostic of the business logic.
It works but the component is not reusable so it sort of defeats the purpose of being a component, try to structure the component so that it's composable and exposes the building blocks that can be used to build a table outside of the component.
As a hint, try to see how component libraries approach this problem.

![Screenshot 2023-02-27 at 12 47 10](https://user-images.githubusercontent.com/39158460/221555926-d94d1f05-9a4b-47ec-872d-6716ba9f3212.png)
![Screenshot 2023-02-27 at 12 47 05](https://user-images.githubusercontent.com/39158460/221555932-e6b129c0-4008-4ecd-a0c3-6f2536c77593.png)